### ISSUE TITLE
Pretrained BERT model?

### ISSUE THREAD

**Issue by priamai** [center-for-threat-informed-defense/tram#197]
Hi there,
I have just deployed the last version via docker and noticed that there are only 2 models pre trained.

![image](https://github.com/center-for-threat-informed-defense/tram/assets/57333254/f21f2cb3-7c26-4937-887b-b5c0387a107e)

It would useful to know how to:
a) train the SCIBERT on some annotated dataset (the link is broken I guess is a private repo)?
b) download a pre-trained SCIBERT

Cheers!
@mehaase

**Comment by mehaase** [comment]
Hi @priamai, that screen is a bit misleading. It is showing stats for the models that were trained inside the container; the SciBERT model is trained outside the container (by us, on high-end GPUs) and downloaded into the docker container. If you want to fine-tune the model on your own data, we have some jupyter notebooks to facilitate that: https://github.com/center-for-threat-informed-defense/tram/wiki/Large-Language-Models#jupyter-notebooks

(I also fixed the broken link that you were looking at: https://github.com/center-for-threat-informed-defense/tram/wiki/Data-Annotation)

**Comment by mehaase** [comment]
Yes it defaults to scibert. The choice of model is specified [`entrypoint.sh`](https://github.com/center-for-threat-informed-defense/tram/blob/main/docker/entrypoint.sh#L53).
